extensions/positron-python/src/client/common/installer/moduleInstaller.ts

```diff
@@ -44,7 +51,16 @@ export abstract class ModuleInstaller implements IModuleInstaller {
         flags?: ModuleInstallFlags,
         options?: InstallOptions,
     ): Promise<void> {
-        const shouldExecuteInTerminal = !options?.installAsProcess;
+        // --- Start Positron ---
+        // python.installModulesInTerminal is a setting that allows the user to force modules to be
+        // installed in the Terminal. Usually, such installations occur in the background. However,
+        // for debugging, it can be helpful to see the Terminal output of the installation process.
+        const workspaceService = this.serviceContainer.get<IWorkspaceService>(IWorkspaceService);
```

Comment:
There's also `IConfigurationService` that is typed, not critical at all.

Reply:
For some reason, when using `IConfigurationService`, I keep getting the default `false` instead of the actual current user setting value. I've now removed the unused ConfigurationService definitions

Reply:
You have to also set the values here: https://github.com/posit-dev/positron/blob/fix-duplicate-notebook-progress-bars/extensions/positron-python/src/client/common/configSettings.ts#L316. I'm not really sure if it's worth it.